eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -86,6 +86,9 @@ parameters:
   - name: PlatformPostSteps
     type: object
     default: []
+- name: TestProxy
```

Comment:
The alignment here is not correct, which causes the live test pipeline failed.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1204102&view=logs&j=96791242-dbf3-587e-3a06-ae5af5c1a705&t=68bc2dc9-cd55-531a-7fab-f5d2306827b9